src/main/kotlin/wu/seal/jsontokotlin/utils/classblockparse/ClassCodeParser.kt

```diff
@@ -93,7 +93,7 @@ class ClassCodeParser(private val classBlockString: String) {
     }
 
     private fun getPropertyKeyword(propertyLine: String): String {
-        val stringBeforeColon = propertyLine.substringBefore(":").trim()
+        val stringBeforeColon = propertyLine.substringBefore("//").substringBeforeLast(":").trim()
```

Comment:
What if the json key contains multiple colons, for instance, "foo:bar:bazz"

Reply:
I have tested that, no error when the json key contains multiple `:`. but I found something exception that the camelcase seems doesn't work well

Reply:
Well then it looks good to me

Reply:
New commit has been push, rename `stringBeforeColon`->`stringBeforeLastColonWithoutComment` to make it clear. And let the name contain `:` could be parsed into camel case